### PR TITLE
Readme css var update

### PR DESCRIPTION
**Pull request by dumblole**
Fixing issue : #1337 (comment)

https://github.com/airbnb/react-dates/pull/1341#pullrequestreview-152672709

### CODE REVIEW

**Comment by dumblole** [comment]
@ljharb Sorry I opened up a new PR, the last one closed after I reverted.

**Comment by dumblole** [comment]
Will this be accepted? If not, please close this issue

File: README.md [hunk 1]
```diff
@@ -74,26 +74,39 @@ By default `react-dates` will use `PureComponent` conditionally if it is availab
...
-.CalendarDay__highlighted_calendar {
-  background: #82E0AA;
-  color: #186A3B;
+// NOTE: the order of these styles DO matter
+
+// Will edit everything selected including everything between a range of dates
+.CalendarDay__selected_span {
+  background: #82e0aa; //background
+  color: white; //text
+  border: 1px solid $light-red; //defualt styles include a border
```
[the remark this answers is not in the document]
Comment (dumblole) [reply README.md]: I'm sorry, did I do something wrong?

[the remark this answers is not in the document]
Comment (ljharb) [reply README.md]: @dumblole “defualt” needs to be “default”

**Comment by dumblole** [comment]
Anything else that's wrong?

**Pull request merged**